### PR TITLE
04 - Create landing page

### PR DESCRIPTION
**Pull request by xDAndrew**
Issue: #4 

- Added static landing page to view articles

### CODE REVIEW

File: GodelTech.CommunityAggregator.UI/src/app/app.component.html [hunk 1]
```diff
@@ -1,20 +1,89 @@
...
-  </li>
-  <li>
-    <h2><a target="_blank" rel="noopener" href="https://github.com/angular/angular-cli/wiki">CLI Documentation</a></h2>
-  </li>
-  <li>
-    <h2><a target="_blank" rel="noopener" href="https://blog.angular.io/">Angular blog</a></h2>
-  </li>
-</ul>
-
+<div id="bg">
```
Comment (akravch) [inline GodelTech.CommunityAggregator.UI/src/app/app.component.html]: It shouldn't all go into a single component. This is what I would like to see:

chrome_2018-08-21_14-48-09

1. `AppComponent`
2. `NavBarComponent`
3. `ArticleComponent`
4. `ArticleListComponent`
5. `FooterComponent`

File: GodelTech.CommunityAggregator.UI/src/styles.css [hunk 2]
```diff
@@ -1 +1,32 @@
...
+	background: #f2f2f2 url('../src/assets/images/bg.png');
+	color: #3d3d3d;
+}
+
+#bg {
+	background: url('../src/assets/images/highlight.png') top left repeat-x;
+	padding: 48px 0 48px 0;
+}
+
+#nav {
```
[the remark this answers is not in the document]
Comment (xDAndrew) [reply GodelTech.CommunityAggregator.UI/src/styles.css]: Let's make a simple website for start.

File: GodelTech.CommunityAggregator.UI/src/styles.css [hunk 3]
```diff
@@ -1 +1,83 @@
...
+
+.imageList {
+	list-style: none;
+}
+
+.imageList li {
+	clear: both;
+}
+
+.bg {
```
Comment (akravch) [inline GodelTech.CommunityAggregator.UI/src/styles.css]: It belongs to the `app-component`, so should be in its style file.

File: GodelTech.CommunityAggregator.UI/src/styles.css [hunk 4]
```diff
@@ -1 +1,83 @@
...
+
+p {
+	margin-bottom: 1.75em;
+}
+
+ul {
+	margin-bottom: 1.75em;
+}
+
+.imageList {
```
Comment (akravch) [inline GodelTech.CommunityAggregator.UI/src/styles.css]: I cannot find where `imageList` or `outer` or `search` classes are used. If we don't use them, please remove the styles.

File: GodelTech.CommunityAggregator.UI/src/app/article-list/article/article.component.css [hunk 5]
```diff
@@ -0,0 +1,23 @@
+.content {
```
Comment (akravch) [inline GodelTech.CommunityAggregator.UI/src/app/article-list/article/article.component.css]: I wouldn't store `article` inside the `article-list` folder. Can we keep them both on the same level?

File: GodelTech.CommunityAggregator.UI/src/app/article-list/article/article.component.html [hunk 6]
```diff
@@ -0,0 +1,13 @@
+<div class="content">
+  <h2>
+    {{ articleData.title }}
+    <div class="publishDate">
+      ({{ articleData.publishDate | date}})
+    </div>
+  </h2>
+  <img class="left" [src]="articleData.imageUrl" width="250" height="250" alt="" />
```
Comment (akravch) [inline GodelTech.CommunityAggregator.UI/src/app/article-list/article/article.component.html]: Might be better to keep width and height in a class.

**Comment by akravch** [comment]
@xDAndrew I'm fine with merging it if we create an issue to make the layout adaptive.

**Pull request merged**